lib/APPReinforcementLearning/src/DrivingState.cpp

```diff
@@ -26,21 +26,17 @@
 *******************************************************************************/
 /**
  * @brief  Driving state
- * @author Andreas Merkle <web@blue-andi.de>
+ * @author Akram Bziouech
  */
 
 /******************************************************************************
  * Includes
  *****************************************************************************/
 #include "DrivingState.h"
-#include <Board.h>
-#include <Sound.h>
-#include <DifferentialDrive.h>
 #include <StateMachine.h>
```

Comment:
Unnecessary change of the order of the includes

Reply:
i have removed the include sound because i don't need it should i reset the order as before or does it fit like this

Reply:
Yes. you can remove the sound without changing the order